html/pfappserver/lib/pfappserver/Form/Config/Provisioning/mobileconfig.pm

```diff
@@ -176,6 +188,16 @@ sub option_security {
     return @security_type;
 }
 
+sub options_conn_type {
+    my $self = shift;
+    my @connection_type = ["wireless" => "Wireless",
+                           "wired" => "Wired",
+                           "wireless-wired" => "Both",
```

Comment:
It doesn't seem there is a template when you're hitting the case where its wireless and wired